bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -23,7 +23,7 @@ const AppContainer = (props) => {
     viewScreenshare,
   } = useSettings(SETTINGS.DATA_SAVING);
   const { isNotificationEnabled } = useReactiveVar(handleIsNotificationEnabled);
-
+  const { showScreenshareQuickSwapButton } = window.meetingClientSettings.public.layout;
```

Comment:
I don't think this prop should be considered.

I think you should check for `hasScreenshareAsContent` instead.
I believe `hasScreenshareAsContent` is always true when the button for quick swap is not enabled, right?